tests/network/probes.go

```diff
@@ -99,7 +98,7 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 				By(specifyingVMReadinessProbe)
 				vmi = createReadyCirrosVMIWithReadinessProbe(readinessProbe)
 
-				assertPodNotReady(virtClient, vmi)
+				Expect(isVMIReady()).To(Equal(false))
```

Comment:
There is one thing I do not like here: `isVMIReady` being a closure that uses the same `vmi` handled by each test.
I need to "trust" it uses the same binded variable and I need to trust that the name/namespace got updates already.

Too much magic and too much assumptions on how the existing code will be used in the future.
Can you also make it a regular function as part of this change?

Reply:
ok, I replaced it, too, with the existing `tests.WaitForVMICondition()` and a new `getVMIConditions()`